packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -0,0 +1,220 @@
+import cdk = require('@aws-cdk/cdk');
+import { CfnVirtualNode } from './appmesh.generated';
+import { HealthCheckProps, ListenerProps, NAME_TAG, PortMappingProps, Protocol } from './shared-interfaces';
+
+// TODO: Add import() and eport() capabilities
```

Comment:
Please pull the comment and instead create GitHub issues for those when the PR is merged.